migrate/v0_13/migrate.go

```diff
@@ -78,6 +89,211 @@ func MigrateCDP(oldGenState v0_11cdp.GenesisState) v0_13cdp.GenesisState {
 	)
 }
 
+// MigrateHard migrates from a v0.11 hard (harvest) genesis state to a v0.13 hard genesis state
+func MigrateHard(genesisState v0_11hard.GenesisState) v0_13hard.GenesisState {
+	v13Deposits := v0_13hard.Deposits{}
+	v13DepositorMap := make(map[string]v0_13hard.Deposit)
+	v13GenesisAccumulationTimes := v0_13hard.GenesisAccumulationTimes{}
+	v13TotalSupplied := sdk.NewCoins()
+
+	for _, dep := range genesisState.Deposits {
+		v13Deposit, ok := v13DepositorMap[dep.Depositor.String()]
+		if !ok {
+			v13Deposit := v0_13hard.NewDeposit(dep.Depositor, sdk.NewCoins(dep.Amount), v0_13hard.SupplyInterestFactors{v0_13hard.NewSupplyInterestFactor(dep.Amount.Denom, sdk.OneDec())})
+			v13DepositorMap[dep.Depositor.String()] = v13Deposit
+		} else {
+			v13Deposit.Amount = v13Deposit.Amount.Add(dep.Amount)
+			v13Deposit.Index = append(v13Deposit.Index, v0_13hard.NewSupplyInterestFactor(dep.Amount.Denom, sdk.OneDec()))
+			v13DepositorMap[dep.Depositor.String()] = v13Deposit
+		}
+	}
+
+	newParams := v0_13hard.NewParams(
+		v0_13hard.MoneyMarkets{
+			// btcb money market - TODO:
+			// 1. is auction size actually used/enforced?
```

Comment:
Looks like `AuctionSize` param isn't used anywhere, should I make an issue/card to remove it?

Reply:
Yes